pkg/cloudprovider/aws/instance.go

```diff
@@ -119,31 +119,26 @@ func (p *InstanceProvider) Create(ctx context.Context,
 	return createFleetOutput.Instances[0].InstanceIds[0], nil
 }
 
-func (p *InstanceProvider) Terminate(ctx context.Context, nodes []*v1.Node) error {
-	if len(nodes) == 0 {
-		return nil
+func (p *InstanceProvider) Terminate(ctx context.Context, node *v1.Node) error {
```

Comment:
What is EC2's delete instance default quota? We're moving from an n -> 1 model for deletion.

Reply:
Since this is in line with how we're doing watch events, I'd like to re-evaluate this when we get to scale testing.

Reply:
FYI It's pretty high for `TerminateInstances` so probably fine but a good thing to watch for when scale testing:

1000 Max and 20 Refill rate

https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-limits